### PR TITLE
Created SQL schema

### LINKED ISSUE #4: Design database tables
3 tables to begin:
- Users
- Pools
- Tickets

### PR DESCRIPTION
**Pull request by antingle**
Closes #4

### CODE REVIEW

**Review by Azure-Agst — COMMENTED** [review]
I'll review types later tonight

**Review by Azure-Agst — APPROVED** [review]
I approve of this, but because I made a bunch of edits, I want at least one other review on here.

**Comment by antingle** [comment]
changed schema to use UUIDs instead of auto increment ints for id fields. Please double check that the syntax is correct

File: schema/schema.sql [hunk 1]
```diff
@@ -20,36 +20,36 @@ DROP TABLE IF EXISTS `lottopool`.`agency`;
...
-    `id` INT NOT NULL AUTO_INCREMENT,
+    `id` UUID NOT NULL,
     `auth0_id` VARCHAR(255) NOT NULL,
     PRIMARY KEY (`id`)
 ) ENGINE=InnoDB CHARSET=utf8 COLLATE=utf8_general_ci;
 
 CREATE TABLE `lottopool`.`agency`(
-    `id` INT NOT NULL AUTO_INCREMENT,
+    `id` UUID NOT NULL,
     `name` BIGINT NOT NULL,
```
Comment (Azure-Agst) [inline schema/schema.sql]: Don't know how this one passed by me. `name` should be VARCHAR.

**Review by Azure-Agst — CHANGES_REQUESTED** [review]
The rest of this works fine, and executes with no errors on the test server.

File: schema/schema.sql (hunk 1 above)
Comment (antingle) [reply schema/schema.sql]: the agency id? wouldn't this be an internal id set by us?

Comment (Azure-Agst) [reply schema/schema.sql]: The ID is fine, it's `name` that I'm concerned about. Preview window isn't the most specific, lol.

Comment (antingle) [reply schema/schema.sql]: ohhhh of course 😂

**Review by Azure-Agst — APPROVED** [review]
LGTM!

**Pull request merged**